scripts/run_integration_tests.sh

```diff
@@ -93,6 +93,21 @@ if [ ! -f "$VIRTIOFSD" ]; then
     popd
 fi
 
+VUBRIDGE="$WORKLOADS_DIR/vubridge"
+VUBRIDGE_DIR="qemu_build"
+if [ ! -f "$VUBRIDGE" ]; then
```

Comment:
There is no need to build qemu twice!

Reply:
To build qemu vhost-user-bridge together with virtiofsd?